File: windows-client/ComponentVault.WinUI/Models/MovementEntryDraft.cs
namespace ComponentVault.WinUI.Models;

public sealed class MovementEntryDraft
{
    public required string ComponentId { get; init; }
    public required string MovementType { get; init; }
    public required int Quantity { get; init; }
    public required string Reason { get; init; }
    public string Note { get; init; } = string.Empty;
}

File: android-client/app/src/main/java/com/componentvault/android/model/MovementEntryDraft.kt
package com.componentvault.android.model

data class MovementEntryDraft(
    val componentId: String,
    val movementType: String,
    val quantity: Int,
    val reason: String,
    val note: String,
    val locationId: String? = null,
    val destinationLocationId: String? = null,
)
